tests/components/anthemav/test_config_flow.py

```diff
@@ -113,3 +115,24 @@ async def test_import_configuration(
         "mac": "00:00:00:00:00:01",
         "model": "MRX 520",
     }
+
+
+async def test_device_already_configured(
+    hass: HomeAssistant,
+    mock_connection_create: AsyncMock,
+    mock_anthemav: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test we import existing configuration."""
+    config = {
+        "host": "1.1.1.1",
+        "port": 14999,
+    }
+
+    mock_config_entry.add_to_hass(hass)
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_IMPORT}, data=config
```

Comment:
Please also test starting a user flow with an already configured device.

Reply:
You mean with a different device? To test that 2 devices can be added?

Reply:
No, I mean a different test case, where the user starts the flow instead of the import starts the flow, ie use `config_entries.SOURCE_USER`.

You can parametrize this with `pytest.mark.parametrize`.

Example:
https://github.com/home-assistant/core/blob/4f5602849105ae9df7441b9a178ad65c1e181833/tests/components/uptime/test_config_flow.py#L38-L52